client/scripts/helpers/snapshot_utils/snapshot_client.ts

```diff
@@ -0,0 +1,6 @@
+import Snapshot from '@snapshot-labs/snapshot.js';
+
+const hubUrl = process.env.SNAPSHOT_APP_HUB_URL || 'https://testnet.snapshot.org';
+const snapshotClient = new Snapshot.Client(hubUrl);
+
+export default snapshotClient;
```

Comment:
This should be added to the snapshot controller so that the client can just be generated once as part of a class

Reply:
rather that it being instantiated each time it's imported

Reply:
Got it